storybook/src/elements/nuxeo-search-form-layout/nuxeo-search-form-layout.stories.js

```diff
@@ -0,0 +1,40 @@
+import { storiesOf } from '@storybook/polymer';
+import { html } from 'lit-html';
+import { LayoutBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-layout-behavior.js';
+import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+import '@nuxeo/nuxeo-ui-elements/search/nuxeo-search-form-layout.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-input.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-aggregation/nuxeo-checkbox-aggregation.js';
+import '../../../.storybook/i18n';
```

Comment:
we're not supposed to have to import this directly iirc 🤔 
perhaps you can move these translations to the messages file, delay this until after the storybook config is loaded (some sort of hook?) or just make the config i18n merge with anything already available in `window.nuxeo.I18n`

Reply:
I just tried to stick as much as possible with [what we already have on `nuxeo-document-layout.stories.js`](https://github.com/nuxeo/nuxeo-elements/pull/207/files#diff-eef68f47ef78b07854aaf14b2778b0e3b448f4ca0d406d893650601c0e104a4bR12), but I can look into it.

Reply:
I'm removing these `import '../../../.storybook/i18n';` lines on both `<nuxeo-search-form-layout>` and `<nuxeo-search-results-layout>` stories, since they don't even seem to make any difference.